cmd/skupper/skupper_kube_service.go

```diff
@@ -91,7 +87,7 @@ func (s *SkupperKubeService) Status(cmd *cobra.Command, args []string) error {
 						theSite := sites.NewChildWithDetail(item, map[string]string{"policy": site.Site.Policy})
 
 						if si.ConnectorCount > 0 {
-							t := mapSiteTargets[site.Site.Identity][si.Name]
+							t := mapSiteTarget[site.Site.Identity][si.Name]
```

Comment:
When I run `skupper service status -v` on the namespace where my headless service has been exposed from (site1), it is not showing the local target.

But if I run it on a linked namespace (site2), then it shows that the service on `site1` provide the respective targets.

Reply:
Nice catch! I could reproduce it and it is fixed in f2f23f1b0cde2c96ab96ffc027480a1c05f1899b Thanks!